mne/defaults.py

```diff
@@ -123,8 +129,10 @@
         fnirs_fd_ac_amplitude=1.0,
         fnirs_fd_phase=1.0,
         fnirs_od=1.0,
-        fnirs_td_gated_amplitude=1e6,
-        fnirs_td_moments_amplitude=1e6,
+        fnirs_td_gated_amplitude=1.0,
+        fnirs_td_moments_intensity=1.0,
+        fnirs_td_moments_mean=1.0,
```

Comment:
the values are in the range of nanoseconds, so this should be 1e-9 for mean and 1e-18 for variance maybe?